app/api/event_copyright.py

```diff
@@ -74,19 +74,17 @@ def before_get_object(self, view_kwargs):
         """
         event = None
         if view_kwargs.get('event_id'):
-            event = safe_query(self, Event, 'id', view_kwargs['event_id'], 'event_id')
+            event = safe_query(Event, 'id', view_kwargs['event_id'], 'event_id')
         elif view_kwargs.get('event_identifier'):
             event = safe_query(
-                self,
                 Event,
                 'identifier',
                 view_kwargs['event_identifier'],
                 'event_identifier',
             )
 
         if event:
-            event_copyright = safe_query(
-                self, EventCopyright, 'event_id', event.id, 'event_id'
+            event_copyright = safe_query(EventCopyright, 'event_id', event.id, 'event_id'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)